src/NzbDrone.Core/Download/ProcessDownloadDecisions.cs

```diff
@@ -73,6 +73,7 @@ public ProcessedDecisions ProcessDecisions(List<DownloadDecision> decisions)
 
                 try
                 {
+                    _logger.Trace("Downloading from Indexer {0} at priority {1}.", remoteEpisode.Release.Indexer, remoteEpisode.Release.IndexerPriority);
```

Comment:
`Grabbing` instead of `Downloading`? Ties in with the naming + I think helps differentiate between the download client downloading and Sonarr passing the release to the client.